Backend/src/config/keys.ts

```diff
@@ -0,0 +1,26 @@
+// types
+interface GoogleKeys {
+  clientID: string;
+  clientSecret: string;
+}
+interface sessionKey {
+  cookieKey: string;
+}
+
+interface Keys {
+  google: GoogleKeys;
+  session: sessionKey;
+}
+const keys: Keys = {
+  google: {
+    clientID:
+      '343654063563-o188120ns26n835285tdq1chb362p375.apps.googleusercontent.com',
+    clientSecret: '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',
+  },
+  session: {
+    cookieKey:

```

Comment:
This key only used for encrypting the cookies, so lets name like that to be easily understood.
```suggestion
    cookieEncryptionKey:

```